src/iteminfo_query.cpp

```diff
@@ -0,0 +1,49 @@
+#include "iteminfo_query.h"
+
+#include <vector>
+#include <string>
+
+
+iteminfo_query::iteminfo_query() : iteminfo_query_base()
```

Comment:
This can be defaulted: `iteminfo_query::iteminfo_query() = default;`

Reply:
👍